addons/stock_account/models/product.py

```diff
@@ -119,6 +118,17 @@ def get_product_accounts(self, fiscal_pos=None):
         })
         return accounts
 
+    def _use_perpetual_valuation(self):
```

Comment:
It's a bit missleading to call this method _use_perpetual_valuation since the base requirement is just realtime + storable + stock valuation acc.

Also it does a lot of override and method just to call it at one place. I would rather update `_get_computed_account` in accrued orders to returns the correct accounts in case of storable realtime